hudi-common/src/main/java/org/apache/hudi/common/table/log/HoodieLogFileReader.java

```diff
@@ -79,6 +79,11 @@
       this.inputStream = fsDataInputStream;
     }
 
+    // Defensive measure to make sure nothing advanced the stream.
+    if (fsDataInputStream.getPos() != 0) {
```

Comment:
@afilipchik : Can we revert this change then and try as it is likely the issue with static vs non-static ?

Reply:
oks